tests/models/gpt2/test_tokenization_gpt2.py

```diff
@@ -250,3 +250,27 @@ def test_add_bos_token_slow(self):
     # tokenizer has no padding token
     def test_padding_different_model_input_name(self):
         pass
+
+
+@require_tokenizers
+class OPTTokenizationTest(unittest.TestCase):
```

Comment:
Out of curiosity, why create a new test class?  :hugs:

Reply:
No mixin, it's a simple test.

If there are better/more suited locations for this test I can move it.
I just thought it didn't fit the Mixin type of tests (which are intended to by highly generic, right ?)

Reply:
Fine for me - think in the normal GPT2 Fast Test we test the tokenizer without leading BOS token. @Narsil could we maybe add 1,2 more tests here to check that fast gives identical output to slow and maybe also a test that the first token (BOS TOKEN) can be changed to whatever token the user wants with save / re-load?

Reply:
I added two tests, are those what you had in mind ?